### DISCUSSION TITLE
Should vckpg_installed directory be added to .gitignore

### DISCUSSION THREAD

**Discussion by perdrix52** [microsoft/vcpkg#42559] [General]
I'm unclear whether this directory that vckpg creates in my project should be checked in to the source code control system (git in my case), or whether it should be added to .gitignore.

Adding the vcpkg.json and vckpg-configuration.json to the repository isn't a big deal, but adding the entirety of the vcpkg-installed directory does appear to be a huge hit.

I'm thinking that if the vcpkg.json looks like this:
```
{
  "dependencies": [
    "libraw",
    "tiff",
    "zlib"
  ],
  "builtin-baseline": "a345bbdc68cdfda65603e24413b21afb28f110fb"
}
```
then if I don't check-in that directory that other project users will just automatically install the pre-requisite packages without burdening the repository with the potentially huge vcpkg-installed directory.

This wasn't discussed in the pdf documentation and IMHO it should be.

**Comment by dg0yt** [comment]
vcpkg-installed isn't source, so it doesn't belong into the source code control system.

In particular, it varies by host, target and other controlling factors.

**Comment by perdrix52** [comment]
Thanks that's the confirmation I needed.
